src/engine/enginemaster.cpp

```diff
@@ -97,6 +101,10 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
+    m_pBoothDelay = new EngineDelay(group, ConfigKey(group, "boothDelay"));
+    m_pLatencyCompensationDelay = new EngineDelay(group,
+        ConfigKey(group, "roundTripLatency"));
```

Comment:
This might be misinterpreted that this CO shows the roundTripLatency, as kind of read only. It also effects only the talkover channel.  
"talkoverDelayCompensation"
"talkoverCompDelay"
"micRoundTripComp"